app/components/agenda/agenda-overview/agenda-overview-item.hbs

```diff
@@ -141,46 +152,6 @@
           </div>
           <div class="vlc-agenda-items__icons">
             <div class="auk-o-flex vlc-agenda-items__icons">
-              {{#if this.loadNewsletterVisibility.lastSuccessful}}
-                {{#if (not this.newsletterIsVisible)}}
-                  <Auk::Icon
-                    @name="not-visible"
-                    @skin="muted"
-                    class="auk-u-ml-2"
-                  >
-                    <EmberTooltip
-                      @side="top-end"
-                      @tooltipClass="tooltip-custom"
-                    >
-                      <p>
-                        {{t "not-visible-in-newsletter"}}
-                      </p>
-                    </EmberTooltip>
-                  </Auk::Icon>
-                {{else}}
-                  {{! TODO: this icon is merely here as an invisible placeholder.
-                    This keeps element-height the same at all times in order to prevent "jumping" when data is being side-loaded.
-                    This should be tackled differently through modified design }}
-                  <Auk::Icon
-                    @name="not-visible"
-                    @skin="invisible"
-                    class="auk-u-ml-2"
-                  />
-                {{/if}}
-              {{else}}
-                {{! TODO: design loading state for newsletter visibility }}
-                <Auk::Icon
-                  @name="circle-question"
-                  @skin="muted"
-                  class="auk-u-ml-2"
-                >
-                  <EmberTooltip @side="top" @tooltipClass="tooltip-custom">
-                    <p>
-                      {{t "loading"}}
-                    </p>
-                  </EmberTooltip>
-                </Auk::Icon>
-              {{/if}}
```

Comment:
Do we still need this workaround to show a temp icon during loading?
Might be hard to test locally, this is a large agenda problem.

Reply:
I would give it a try without the loading state and await user's feedback. If a loading icon must be shown, we should use another one than `circle-question` since that's also used for an undetermined formally OK status at the same location. This might confuse users.